ui/pages/SummaryData/constants.js

```diff
@@ -1,9 +1,7 @@
-/* eslint-disable import/prefer-default-export */
-
```

Comment:
There is a warning that the `eslint-disable` doesn't change anything because there are no `import/prefer-default-export` violations.